src/pages/index.js

```diff
@@ -1,62 +1,75 @@
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
 
-export default function Home() {
+
+
+function Home() {
   return (
-    <Box sx={{
-      maxWidth: "1200px",
-      mx: 'auto',
-    }}>
-      {/* Top Header */}
-      <Box sx={{ display: 'flex', gridGap: '1rem', pt: '1.5rem' }}>
-        <StaticImage width={60} src={'../assets/logo.png'} />
-        <Box component='header' sx={{
-          fontWeight: 'bold',
-          fontSize: "2.2rem",
-          fontFamily: "sans-serif",
-          px: '1rem'
-        }}>
-          Capstone
-        </Box>
-      </Box>
-
-      <Box sx={{
-        mt: "5rem",
-        px: "2rem",
-      }}>
-        {/* Top Heading */}
-        <Box component={'h1'} sx={{
-          fontSize: '3rem'
-        }}>
-          Thoth Tech
-        </Box>
-        <Box sx={{
-          fontSize: '1.2rem',
-        }}>
-          One step for all projects
-        </Box>
-        <Box sx={{
-          mt: "2rem"
-        }}>
-        <Box component={Link} to={'/docs'} sx={{
-            background: "#000",
-            color: '#fff',
-            padding: '8px 1rem',
-            fontSize: '1.2rem',
-            textDecoration: "none",
-            borderRadius: '12px',
-            "&hover": {
-              background: "#000"
-            }
-          }} variant='contained'>
-            Explore Docs
-          </Box>
-        </Box>
-      </Box>
-
-    </Box>
-  )
+    <Container>
+      <Navbar className="navbar-color" bg="grey" expand="lg">
+        
+
+          <Navbar.Brand href="/">
+            <StaticImage className="brandimg" width={40} src={'../assets/logo.png'} />
+          </Navbar.Brand>
+          <Navbar.Brand href="#home">
+            Capstone
+          </Navbar.Brand>
+
+          <Navbar.Toggle aria-controls="basic-navbar-nav" />
+          <Navbar.Collapse id="basic-navbar-nav">
+
+            <Nav className="justify-content-end" style={{width: "100%"}}>
+
+              <Nav.Link>
+                <Button className="navlink" href="#home" variant="outline-dark">Home</Button>
+              </Nav.Link>
+              <Nav.Link>
+                <Button className="navlink" href="#products" variant="outline-dark">Products</Button>
+              </Nav.Link>
+              <Nav.Link>
+                <Button className="navlink" href="#getstarted" variant="outline-dark">Get Started</Button>
+              </Nav.Link>
+
+            </Nav>
+        </Navbar.Collapse>
+      </Navbar>
+
+      <Row className="row1">
+        <Col className="col1">
+          <div className="block1">
+            <h1 className="heading">
+              Thoth Tech
+            </h1>
+            <br></br>
+            <p className="para">One stop for all projects</p>
+            
+            <Button className="btn" variant="dark">
+              <Box className="link" component={Link} to={'/docs'} variant='contained'>
+                Get Started
```

Comment:
Instead of get started, please put "Explore Docs", since we are just exploring documenation here